### ISSUE TITLE
Cannot read properties of undefined (reading 'mover') when clicks on tab item

### ISSUE THREAD

**Issue by shaohaolin** [microsoft/tabster#160]
We are using the tablist component from `@fluentui/react-components` version 9.0.1.  `@fluentui/react-tabster` requires `tabster@^1.4.2` ([package.json](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-tabster/package.json#L39)); note the caret which permits upgrades to minor versions.

We start seeing the following error after upgrading tabster to 1.5.1.
![image](https://user-images.githubusercontent.com/6677092/177222836-dba09571-e27f-4e35-90a2-ffde23bea929.png)

This error happens when a tab item is clicked. This is due to the `useTabster` hook in [@fluentui](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-tabster/src/hooks/useTabster.ts#L22) calls the ` getCurrentTabster` or `createTabster` methods.
```js
export const useTabster = (): TabsterTypes.TabsterCore | null => {
  ...
  return getCurrentTabster(defaultView) ?? createTabster(defaultView, tabsterOptions);
};

```
The `getCurrentTabster` returns an instance of `TabsterCore` and the `createTabster` returns an instance of `Tabster`.  `createTabster()` method assigning `TabsterCore` instance to the windows object instead of the `Tabster` instance. See code below:
```js
/**
 * Creates an instance of Tabster, returns the current window instance if it already exists.
 */

function createTabster(win, props) {
  let tabster = getCurrentTabster(win);

  if (tabster) {
    return tabster.createTabster();
  }

  tabster = new TabsterCore(win, props);
  win.__tabsterInstance = tabster;
  return tabster.createTabster();
}
```
When the `getMover` method is trying to access `tabster.core`, `tabster.core` is undefined. 

Proposal change: assigning `Tabster` instance to windows and return it.
```js
function createTabster(win, props) {
  ...
  let tabsterCore = new TabsterCore(win, props);
  tabster = tabsterCore.createTabster();
  win.__tabsterInstance = tabster;
  return tabster;
}
```

Another proposal change in Fluent UI side, change the return in the `useTabster` hook to call `createTabster` method only.
```js
export const useTabster = (): TabsterTypes.TabsterCore | null => {
  ...
  return createTabster(defaultView, tabsterOptions);
};
```

**Comment by ling1726** [comment]
This was fixed since the 1.5.4 release when the entire offending PR was reverted and moved to 2.0.0 https://github.com/microsoft/tabster/blob/master/CHANGELOG.md#v154

**Comment by shaohaolin** [comment]
Great, thank you @ling1726
